src/cli/setup/aggregate.rs

```diff
@@ -0,0 +1,118 @@
+use crate::cli::setup::index_of;
+use crate::cli::setup::traits::Response;
+use crate::crypto::vss::Vss;
+use crate::errors::Error;
+use crate::net::SignerID;
+use crate::sign::Sign;
+use crate::signer_node::SharedSecret;
+use crate::signer_node::SharedSecretMap;
+use bitcoin::{PrivateKey, PublicKey};
+use clap::{App, Arg, ArgMatches, SubCommand};
+use curv::cryptographic_primitives::secret_sharing::feldman_vss::ShamirSecretSharing;
+use curv::cryptographic_primitives::secret_sharing::feldman_vss::VerifiableSS;
+use curv::elliptic::curves::traits::ECPoint;
+use curv::elliptic::curves::traits::ECScalar;
+use curv::FE;
+use std::fmt;
+use std::str::FromStr;
+
+pub struct AggregateResponse {
+    aggregated_public_key: PublicKey,
+    node_shared_secret: FE,
+}
+
+impl AggregateResponse {
+    fn new(aggregated_public_key: PublicKey, node_shared_secret: FE) -> Self {
+        AggregateResponse {
+            aggregated_public_key: aggregated_public_key,
+            node_shared_secret: node_shared_secret,
+        }
+    }
+}
+
+impl Response for AggregateResponse {}
+
+impl fmt::Display for AggregateResponse {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "{} {}",
+            self.aggregated_public_key,
+            self.node_shared_secret.to_big_int(),
```

Comment:
This BigInt would display as decimal number, right? I think it is better to show as 32bytes hex string.

Reply:
fixed at 158dad828a182b82d923837036e619e1f2066484